src/freenet/support/HTMLNode.java

```diff
@@ -418,7 +418,7 @@ public StringBuilder generate(StringBuilder tagBuffer, int indentDepth ) {
 			}
 		} else {
 			if (newlineOpen(name)) {
-				tagBuffer.append('\n');
+				tagBuffer.append(System.lineSeparator());
```

Comment:
Here it’s basically the same reasoning: we can’t actually know that the HTML file is being parsed on the same OS that we use to create the HTML.